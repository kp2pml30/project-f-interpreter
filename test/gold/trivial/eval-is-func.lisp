(setq me eval)
(setq a 11)
(setq b 12)
(setq exp (quote (plus a b)))
(setq a 3)
(setq b 27)
(eval 11)
(eval exp)
(me exp)
