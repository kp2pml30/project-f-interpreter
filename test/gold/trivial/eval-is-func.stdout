"int:11"
"int:30"
"int:30"
